.devcontainer.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Is there a reason for having the devcontainer both at root and in the .devcontainer folder? I will probably remove this one and keep the nested version in ./.devcontainer if not